src/main/webapp/components/SampleCollectionWizard.js

```diff
@@ -0,0 +1,133 @@
+import { Component } from 'react';
+import { div, hh, label, span } from 'react-hyperscript-helpers';
+import { SecurityReview } from "./SecurityReview";
+import { IntCohortsReview } from "./IntCohortsReview";
+import { isEmpty } from "../util/Utils";
+import { Documents } from "./Documents";
+import { Table } from "./Table";
+
+const headers =
+  [
+    { name: 'Document Type', value: 'fileType' },
+    { name: 'File Name', value: 'fileName' }
+  ];
+
+export const SampleCollectionWizard = hh(class SampleCollectionWizard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentStepIndex: 0,
+    };
+  }
+
+  componentDidCatch(error, info) {
+    console.log('----------------------- error ----------------------');
+    console.log(error, info);
```

Comment:
Note for @vvicario that her current error story will be impacted by this.

Reply:
sure, I will remove it. I already removed them everywhere in my current branch

Reply:
I'm not asking for this to be removed in this PR, just noting for the record that your error story will need to look at this instance too.